scripts/common/host-packages.sh

```diff
@@ -114,6 +114,13 @@ function _dpkg_install_host_packages() {
         return 0
     fi
 
+    # Remove docker packages, if installed to install containerd
```

Comment:
```suggestion
    # Remove docker packages, if installed to install containerd
    # It is required because of an bug where the package manager is unable to sort it out and 
    # the installation/upgrade will fails with `dpkg: no, cannot proceed with removal of containerd ... docker.io     
    # depends on containerd (>= 1.2.6-0ubuntu1~)  containerd is to be removed.` 
    # More info: https://bugs.launchpad.net/ubuntu/+source/docker.io/+bug/1940920
    # https://bugs.launchpad.net/ubuntu/+source/docker.io/+bug/1939140       
```